end-to-end-testing-helpers/index.ts

```diff
@@ -100,13 +100,13 @@ export function hover(elem: ElementFinder) {
 
 export function selectFilesForUpload(
     fileInput: ElementFinder,
-    fileNames: Array<string>, // relative to spec/test-files folder
+    fileNames: Array<string>, // relative to e2e/client/specs/test-files folder
 ) {
     fileInput.sendKeys(
         fileNames
             .map(
                 (relativePath) =>
-                    path.resolve(__dirname, '../../../../spec/test-files/' + relativePath),
+                    path.resolve(__dirname, '../../../../../../e2e/client/specs/test-files/' + relativePath),
```

Comment:
going 6 levels up looks a bit strange, shouldn't it get some base path as argument rather?

Reply:
I agree it is a bit strange. I could have used less levels, but I think it's more clear if I go all the way up to the root path. Since we now have a dedicated npm package for e2e tests exclusively, I think I could remove `end-to-end-testing-helpers` as a separate package and move it to e2e dir. The path would be simpler and we wouldn't need to bother with publishing to npm and versioning. What do you think?

Reply:
yep moving that to e2e sounds good

Reply:
In the end I didn't move it, because extensions will and do need the helpers and they are not supposed to import things outside their extension directory. I have changed the function so it takes an absolute and I don't need to go 6 level up (which would not have worked when used from extensions anyway).